### PR TITLE
[13.0][MIG] purchase_product_usage

### PR DESCRIPTION
**Pull request by kongrattapong**
Ready to review

### CODE REVIEW

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional review LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

File: purchase_product_usage/models/purchase_product_usage.py [hunk 1]
```diff
@@ -30,15 +30,16 @@ class ProductUsage(models.Model):
...
-        new_result = []
-        for usage in result:
-            rec = self.browse(usage[0])
-            if rec.code:
-                name = "[{}] {}".format(rec.code, usage[1])
-            else:
-                name = usage[1]
-            new_result.append((rec.id, name))
-        return new_result
+        for rec in self:
```
Comment (HviorForgeFlow) [inline purchase_product_usage/models/purchase_product_usage.py]: I don't understand this change, code does not seems ok to me.
@LoisRForgeFlow @AdriaGForgeFlow

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: purchase_product_usage/models/purchase_product_usage.py (hunk 1 above)
Comment (kongrattapong) [reply purchase_product_usage/models/purchase_product_usage.py]: @HviorForgeFlow at first, I think it's might be multi record but it's seem not necessary. so I'll change to original

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**